src/Abp/RealTime/IOnlineClientStore.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using Abp.Data;
+
+namespace Abp.RealTime
+{
+    public interface IOnlineClientStore
+    {
+        /// <summary>
+        /// Adds a client.
+        /// </summary>
+        /// <param name="client">The client.</param>
+        void Add(IOnlineClient client);
+
+        /// <summary>
+        /// Removes a client by connection id.
+        /// </summary>
+        /// <param name="connectionId">The connection id.</param>
+        /// <returns>Tuple indicating whether the client was removed and if so, the value</returns>
+        ConditionalValue<IOnlineClient> Remove(string connectionId);
```

Comment:
Seems like a good candidate to implements `TryGet` and `TryRemove` here instead of using `ConditionalValue`

Reply:
https://github.com/aspnetboilerplate/aspnetboilerplate/pull/4411